### ISSUE TITLE
Revert  GO MIRROR in cl.Makefile

### ISSUE THREAD

**Issue by matentzn** [obophenotype/cell-ontology#1378]
```
mirror/go.owl: mirror/go.trigger
	echo "WARNING OVERWRITING GO MIRROR BECAUSE OF OBSOLETE CL TERM"
	if [ $(MIR) = true ] && [ $(IMP) = true ]; then curl -L $(URIBASE)/go/go-base.owl --create-dirs -o mirror/go.owl --retry 4 --max-time 200 && $(ROBOT) remove -i mirror/go.owl --term "CL:0000243" --preserve-structure false convert -o $@.tmp.owl && mv $@.tmp.owl $@; fi
.PRECIOUS: mirror/go.owl
```

Should be reverted when GO replaces `CL:0000243`.

@rays22 can you check if http://purl.obolibrary.org/obo/go/go-base.owl still uses CL:0000243? If so, make sure there is an issue on the GO repo and link it here.

**Comment by dosumis** [comment]
I thought the fix would be to RO ?

**Comment by dosumis** [comment]
OK. I can see this is a separate issue. Please ignore my last comment.

**Comment by rays22** [comment]
> @rays22 can you check if http://purl.obolibrary.org/obo/go/go-base.owl still uses CL:0000243? If so, make sure there is an issue on the GO repo and link it here.

I can confirm that CL:0000243 is not used by [go-basic.obo.](http://purl.obolibrary.org/obo/go/go-basic.obo )

**Comment by matentzn** [comment]
Ok thanks @rays22 , @shawntanzk whenever you have time (after RO PR is merged), you can deal with this, but no rush at all.

**Comment by shawntanzk** [comment]
I assume the RO PR mentioned is this: https://github.com/oborel/obo-relations/pull/543

**Comment by matentzn** [comment]
For this issue here, its only about the obsolete class. If the class is no longer obsolete, you can jus remove the above from cl.Makefile and regenerate the import

**Comment by matentzn** [comment]
@shawntanzk is this still an issue?

**Comment by shawntanzk** [comment]
looks like all PRs related are merged and i assume GO would have made a release by now, happy to give it a go and see if its all good

**Comment by shawntanzk** [comment]
actually let me check go-base first

**Comment by shawntanzk** [comment]
Merged master into #1388 and saw the change in import removing the obsolete CL term - will let checks run and if all good, ill merge it in
